rdrf/rdrf/views/patient_view.py

```diff
@@ -539,22 +547,44 @@ def get_forms(self, request, registry_model, user, instance=None):
 
         return forms
 
-    def _section_hidden(self, user, registry, fieldlist):
+    def _exclude_hidden_fields(self, user, registry, fieldlist):
+        from rdrf.models.definition.models import DemographicFields
+        user_groups = [g.name for g in user.groups.all()]
```

Comment:
I don't think you have to get the group names. Just filter by `groups__in=user.groups.all()` instead.
Also applies to the other 2 "hidden" methods.

Reply:
Updated.